python_files/data_generator.py

```
@@ -0,0 +1,539 @@
+                    'nameOrig': 'ID_Source',
+                    'oldbalanceOrg': 'Old_Balance',
+                    'newbalanceOrig': 'New_Balance',
+                    'nameDest': 'ID_Dest',
+                    'isFraud': 'Is_Fraud'
+                }, inplace=True)
+            except FileNotFoundError:
+                print("Path not found!")
+        elif self.data_name == 'fraud':
+            try: 
+                self.df = None
+            except Exception as e:
```

Ruff (0.8.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)